amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/.gitignore

```diff
@@ -0,0 +1,2 @@
+.synapseConfig
```

Comment:
This is here because I wanted a way to refer to a `.synapseConfig` that does not depend on a particular user's path.